### PR TITLE
Update Deps to work for Storybook 7.0

### PR DESCRIPTION
**Pull request by Integrayshaun**
Addresses #7

## What Changed
- [x] Move `@storybook/*` from `peerDependencies` to `dependencies`
- [x] Upgraded `@storybook/*` to 7.0
- [x] Updated imports for some storybook dependencies
- [x] Upgraded CI node image to node 16

## How it was tested
- [x] Run on external storybook project @7.0
- [ ] Run on external storybook project @6.5

<details>
  <summary>📦 Published PR as canary version: <code>1.0.0-canary.8.e2cc917.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/addon-styling@1.0.0-canary.8.e2cc917.0
  # or 
  yarn add @storybook/addon-styling@1.0.0-canary.8.e2cc917.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,20 +37,31 @@
...
-    "start": "concurrently \"yarn build:watch\" \"yarn storybook -- --no-manager-cache --quiet\"",
-    "build-storybook": "build-storybook",
+    "storybook": "storybook dev -p 6006",
+    "start": "concurrently \"yarn build:watch\" \"yarn storybook -- --quiet\"",
+    "build-storybook": "storybook build",
     "prerelease": "zx scripts/prepublish-checks.mjs",
     "release": "yarn build && auto shipit"
   },
+  "dependencies": {
+    "@storybook/addons": "^7.0.0-rc.3",
```
Comment (yannbf) [inline package.json:47]: Doesn't seem like you need this

File: package.json [hunk 2]
```diff
@@ -37,20 +37,31 @@
...
+    "@storybook/api": "^7.0.0-rc.3",
+    "@storybook/components": "^7.0.0-rc.3",
+    "@storybook/core-events": "^7.0.0-rc.3",
+    "@storybook/manager-api": "^7.0.0-rc.3",
+    "@storybook/preview-api": "^7.0.0-rc.3",
+    "@storybook/theming": "^7.0.0-rc.3",
+    "@storybook/types": "^7.0.0-rc.3"
```
Comment (yannbf) [inline package.json:54]: I'm very curious about this change, did the addon not have any dependencies at all?

Comment (Integrayshaun) [reply package.json:54]: They were in peerdep before

**Comment by Integrayshaun** [comment]
Closed in favour of prerelease branch strategy

**Pull request closed without merging**